scss/components/_form-styles.scss

```diff
@@ -219,3 +219,61 @@
 .date-range-input {
   padding-left: u(2.5rem);
 }
+
+// Loading states
+label {
+
+  &::after {
+    background: transparent;
+    content: '';
+    display: inline-block;
+    width: 2rem;
```

Comment:
I think this is my fault, but these should all be `u(...)`. This is necessary because the styles are shared by the FEC instance of eRegs.